### PR TITLE
[issue_4557] Fix alignment issues in the error state switch

### PR DESCRIPTION
**Pull request by d29107d**
## Summary
Relate to https://github.com/highlight/highlight/issues/4557

## How did you test this change?
Local UI is ok.
image

## Are there any deployment considerations?
No

### CODE REVIEW

**Comment by d29107d** [comment]
@Vadman97 
Could you give it a review?

**Review by Vadman97 — APPROVED** [review]
looks good! CI failing due to lint check (run `yarn format:all` then commit the changes)

**Comment by d29107d** [comment]
@Vadman97 
I recommit the code, could you help to rerun the CI?

File: frontend/src/pages/ErrorsV2/ErrorStateSelect/ErrorStateSelect.tsx
Comment (et) [inline frontend/src/pages/ErrorsV2/ErrorStateSelect/ErrorStateSelect.tsx:354]: @d29107d - your editor is removing the newline here and that's why the build is failing.
Per what @Vadman97 said, can you run `yarn format:all` and commit the change?

Comment (d29107d) [reply frontend/src/pages/ErrorsV2/ErrorStateSelect/ErrorStateSelect.tsx]: Done. 
@et could you help to rerun the CI?

Comment (et) [reply frontend/src/pages/ErrorsV2/ErrorStateSelect/ErrorStateSelect.tsx]: Yup. Just ran it.

Comment (d29107d) [reply frontend/src/pages/ErrorsV2/ErrorStateSelect/ErrorStateSelect.tsx]: @et I don't know why the CI failed so I rebase the main branch, it needs you to rerun the CI. 🙏

Comment (et) [reply frontend/src/pages/ErrorsV2/ErrorStateSelect/ErrorStateSelect.tsx]: The error seems like it might have been related to #4703. Rebasing definitely seems like the right call.
Just re-ran. Hopefully this one passes 🤞

Comment (et) [reply frontend/src/pages/ErrorsV2/ErrorStateSelect/ErrorStateSelect.tsx]: Actually. It looks like we might have to wait until #4734 is merged through. Will let you know when that happens. Thanks for your work on this @d29107d

Comment (d29107d) [reply frontend/src/pages/ErrorsV2/ErrorStateSelect/ErrorStateSelect.tsx]: 👌

**Pull request merged**
